x/clp/client/rest/rest.go

```diff
@@ -0,0 +1,15 @@
+package clp
+
+import (
+	"github.com/cosmos/cosmos-sdk/crypto/keys"
+	"github.com/gorilla/mux"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/wire"
+)
+
+// RegisterRoutes registers staking-related REST handlers to a router
+func RegisterRoutes(ctx context.CoreContext, r *mux.Router, cdc *wire.Codec, kb keys.Keybase) {
+	registerQueryRoutes(ctx, r, cdc)
+	registerTxRoutes(ctx, r, cdc, kb)
```

Comment:
Don't want to be pedantic, but it might help to name the files a bit closer to what what they contain. The generic `registerTxRoutes` function comes from the specifically called `sendtx.go` file. What do you think?

Reply:
Yeh i was following cosmos conventions (which suck and are inconsistent anyway lol), agree file should just be tx.go

Reply:
👍